lib/node_modules/@stdlib/blas/base/csscal/README.md

```diff
@@ -0,0 +1,174 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# csscal
+
+> Scale a single-precision complex floating-point vector by a single-precision floating-point constant.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var csscal = require( '@stdlib/blas/base/csscal' );
+```
+
+#### csscal( N, sa, cx, strideZX )
+
+Scales a single-precision complex floating-point vector by a single-precision floating-point constant.
+
+```javascript
+var Complex64Array = require( '@stdlib/array/complex64' );
+
+var cx = new Complex64Array( [ 1.0, 1.0, 1.0, 1.0, 1.0, 1.0 ] );
```

Comment:
```suggestion
var x = new Complex64Array( [ 1.0, 1.0, 1.0, 1.0, 1.0, 1.0 ] );
```

Here and everywhere so that we are consistent in variable naming with the real-valued implementations.